app/helpers/projects_helper.rb

```diff
@@ -0,0 +1,29 @@
+module ProjectsHelper
+  def project_teams(project)
+    Team.where.not(id: project.teams.select(:id)) 
+  end
+
+  def isTeamAvailable?(project)
+    Team.where.not(id: project.teams.select(:id)).to_a.empty?
+  end
+
+  def project_display_name(project)
+    display_name = nil
+    if project.project_category == 'TEAM'
```

Comment:
Create a constant for project categories in Project model and use that wherever needed.